BrightID/src/components/NewConnectionsScreens/PreviewConnectionScreen.js

```diff
@@ -27,155 +30,163 @@ import api from '../../Api/BrightId';
-      console.log('flaggers', flaggers);
-      const mutualConnections = connections.filter(function (el) {
-        return myConnections.some((x) => x.id === el.id);
-      });
-      this.setState({
-        connections: connections.length,
-        groups: groups.length,
-        mutualConnections: mutualConnections.length,
-        connectionDate: `Created ${moment(parseInt(createdAt, 10)).fromNow()}`,
-        flagged: flaggers && Object.keys(flaggers).length > 0,
-      });
-    } catch (err) {
-      if (err instanceof Error && err.message === 'User not found') {
-        this.setState({
-          connections: 0,
-          groups: 0,
-          mutualConnections: 0,
-          connectionDate: 'New user',
-          flagged: false,
-        });
-      } else {
-        err instanceof Error ? console.warn(err.message) : console.log(err);
-      }
-    }
-  };
-
-  render() {
-    const {
-      connectUserData: { photo, name },
-    } = this.props;
-    const image = photo ? { uri: photo } : { uri: 'https://picsum.photos/180' };
-    return (
-      <SafeAreaView style={styles.container} testID="previewConnectionScreen">
-        <StatusBar
-          barStyle="dark-content"
-          backgroundColor="#fff"
-          translucent={false}
-          animated={true}
+  return (
+    <SafeAreaView style={styles.container} testID="previewConnectionScreen">
+      <StatusBar
+        barStyle="dark-content"
+        backgroundColor="#fff"
+        translucent={false}
+        animated={true}
+      />
+      <View style={styles.questionTextContainer}>
+        <Text style={styles.questionText}>Connect with?</Text>
+      </View>
+      <View style={styles.userContainer}>
+        <Image
+          source={{ uri: connectUserData.photo }}
+          style={styles.photo}
+          resizeMode="cover"
+          onError={(e) => {
+            console.log(e);
+          }}
+          accessible={true}
+          accessibilityLabel="user photo"
         />
-        <View style={styles.questionTextContainer}>
-          <Text style={styles.questionText}>Connect with?</Text>
-        </View>
-        <View style={styles.userContainer}>
-          <Image
-            source={image}
-            style={styles.photo}
-            resizeMode="cover"
-            onError={(e) => {
-              console.log(e);
-            }}
-            accessible={true}
-            accessibilityLabel="user photo"
-          />
-          <Text style={styles.connectName}>
-            {name}
-            {this.state.flagged && (
-              <Text style={styles.flagged}> (flagged)</Text>
-            )}
+        <Text style={styles.connectName}>
+          {connectUserData.name}
+          {userInfo.flagged && <Text style={styles.flagged}> (flagged)</Text>}
+        </Text>
+        <Text style={styles.connectedText}>{userInfo.connectionDate}</Text>
+      </View>
+      <View style={styles.countsContainer}>
+        <View style={styles.countsGroup}>
```

Comment:
Probably not related to this PR, but styles.countsGroup seems to be undefined.
Same undefined style appears 2 more times below.

Reply:
I saw the same warning, we should probably remove those style objects. I didn't style this component so I just left it alone